src/ShopBundle/objects/db/TShopArticle.class.php

```diff
@@ -1907,6 +1907,8 @@ public function UpdateStock($dNewStockValue, $bNewAmountIsDelta = false, $bUpdat
             return false;
         }
         $oldStock = $this->getAvailableStock();
+        //$valueChanged = (double)$oldStock !== (double)$dNewStockValue;
+        // NOTE the below comparisson always has true as result (compares int to double); see #84
```

Comment:
*comparison

When referring to issues, I'd like to add complete links/URLs so that a reader doesn't have to search for the issue tracker. Also I added an issue to address the data type of the stock value: https://github.com/chameleon-system/chameleon-system/issues/120 (to give it more space than in the combined issue #83). Could you link that one?